packages/jest-validate/src/validate_cli_options.js

```diff
@@ -80,12 +80,12 @@ export default function validateCLIOptions(argv: Argv, options: Object) {
     throw createCLIValidationError(unrecognizedOptions, allowedOptions);
   }
 
-  const CLIDeprecations = Object.keys(deprecationEntries).reduce(
+  const CLIDeprecations = Object.keys(CLIDeprecationEntries).reduce(
```

Comment:
I think `jest-config` should pass `deprecationEntries` into `jest-validate`, they don't have to live in `jest-validate`. If I use `jest-validate` for config validation in my own module, I don't care about jest's deprecations

Reply:
@SimenB that was the previous behavior... But caused the deadlock.

Reply:
BTW, I renamed to `CLIDeprecationEntries` because there's another variable with the similar name. It was confusing.

Reply:
jest-validate imported it from jest-config - I think jest-config should explicitly pass it to jest-validate

Reply:
@SimenB make sense